docs/dev-tools/.pages

```diff
@@ -0,0 +1,2 @@
+nav:
```

Comment:
there will need to be an index.md file for Dev Tools. once you merge the new changes in from `main`, you can check out an example under the `testnets` directory. It'll need to have `template: subsection-index-page.html`

Reply:
I see, I will fix that